source/atlas-cli-tutorials.txt

```diff
@@ -0,0 +1,46 @@
+.. _atlas-cli-tutorials:
+
+=======================================
+Manage |service| from the {+atlas-cli+}
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use the following tutorials to learn how to manage |service| using the 
+{+atlas-cli+}. 
+
+To learn the {+atlas-cli+} basics, see :ref:`install-atlas-cli` and
+:ref:`connect-atlas-cli`.
+
+You can also go straight to the :doc:`{+atlas-cli+} Commands 
+</command/atlas>`.
+
+.. list-table:: 
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Tutorial
+     - Objective
+
+   * - :ref:`atlas-cli-onboarding`
+     - Use the ``atlas setup`` command to onboard with |service|,
+       including creating your account and {+cluster+}. You can also
+       use this tutorial to create your first {+cluster+} even if you
+       already have an |service| account.
+
+   * - :ref:`atlas-cli-quick-start`
```

Comment:
Can we rename the tutorial for quickstart?

[Set Up Atlas with atlas quickstart](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-22576-2/atlas-cli-quickstart/#std-label-atlas-cli-quick-start)

Set Up and quickstart in the same phrase seems confusing.

Reply:
Good point. I renamed it. Thanks!